.teamcity/src/main/kotlin/model/CIBuildModel.kt

```diff
@@ -58,7 +59,7 @@ data class CIBuildModel(
         Stage(
             StageName.QUICK_FEEDBACK_LINUX_ONLY,
             specificBuilds = listOf(
-                SpecificBuild.CompileAll, SpecificBuild.SanityCheck
+                SpecificBuild.CompileAll, SpecificBuild.CompileAllBuildCacheNG, SpecificBuild.SanityCheck
```

Comment:
Could you move this to a (new) experimental stage and not put it in quick feedback linux at all? Then you also don't need to deal with setting different dependencies I suppose.